src/Http/Http/ref/Microsoft.AspNetCore.Http.netcoreapp3.0.cs

```diff
@@ -195,6 +195,26 @@ public static partial class RequestFormReaderExtensions
     {
         public static System.Threading.Tasks.Task<Microsoft.AspNetCore.Http.IFormCollection> ReadFormAsync(this Microsoft.AspNetCore.Http.HttpRequest request, Microsoft.AspNetCore.Http.Features.FormOptions options, System.Threading.CancellationToken cancellationToken = default(System.Threading.CancellationToken)) { throw null; }
     }
+    public static partial class SendFileFallback
```

Comment:
This name kinda sucks...

Reply:
SendFileHelpers?
SendFileStreamAdapter?

Reply:
Does it really need to be public? I think shared source would be more appropriate and then the name doesn't matter as much.

Reply:
Yes it should be public, any implementer of IHttpResponseBodyFeature is likely to need it (e.g. ResponseCompression).